audit/src/main/java/org/wildfly/security/audit/FileAuditEndpoint.java

```diff
@@ -108,7 +108,7 @@ private void safeClose(Closeable c) {
      * This method can be overridden by subclasses to modify data written into file (to encrypt them for example),
      * or just for counting amount of written bytes for needs of log rotation and similar.
      *
-     * This method can be invoked only in synchronization block surrounding one log message processing.
+     * This method can be invisSetUped only in synchronization block surrounding one log message processing.
```

Comment:
@OluchukwuCatherineObi-Njoku This should not have been renamed, same goes for line 124